qa/standalone/ceph-helpers.sh

```diff
@@ -22,6 +22,7 @@ WAIT_FOR_CLEAN_TIMEOUT=90
 MAX_TIMEOUT=15
 PG_NUM=4
 CEPH_BUILD_VIRTUALENV=${TMPDIR:-/tmp}
+ARCHIVE_DIR=${TESTDIR:-${CEPH_BUILD_VIRTUALENV}}
```

Comment:
i'd suggest change it to
```sh
TESTDIR=${TESTDIR:-/tmp}"
```

as it's misleading that we want to use `$CEPH_BUILD_VIRTUALENV` for archiving logging messages.

Reply:
@tchaikov reason I did it this way is that it allows the user still to define `TMPDIR` and the archive will land where it was expected.
I'm not sure if one can nest this construction, but otherwise it would be:
`TESTDIR=${TESTDIR:-${TMPDIR:-/tmp}}` 
I'll check